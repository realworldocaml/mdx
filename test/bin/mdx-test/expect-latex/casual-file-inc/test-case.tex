Mdx can also include text from any file:


% $MDX file=casual-file.txt
\begin{ocaml}
\end{ocaml}
