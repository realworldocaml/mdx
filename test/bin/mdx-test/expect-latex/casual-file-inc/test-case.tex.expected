Mdx can also include text from any file:


% $MDX file=casual-file.txt
\begin{ocaml}
Include me.

Please.
\end{ocaml}
